### PR TITLE
[RFC] Added namespace to transitionEndEvents in waitForTransition - turn off t...

### PR DESCRIPTION
**Pull request by abhisheksavant**
@jskulski

### CODE REVIEW

File: test/test-waitForTransition.js [hunk 1]
```diff
@@ -18,24 +18,43 @@ buster.testCase('waitForTransition',
...
+      , callback = this.spy()
+      ;
+
+    this.testObject.waitForTransition(target, callback);
+
+    var transitionEvents = target.one.args[0][0];
+
+    target.one.getCall(0).callArg(1);
+    // assert off is called once with transition events
+    assert.calledOnceWith(target.off, transitionEvents);
```
Comment (alexindigo) [inline test/test-waitForTransition.js:43]: Can you trigger that event one more time and make it it still called only once?

File: test/test-waitForTransition.js [hunk 2]
```diff
@@ -18,24 +18,43 @@ buster.testCase('waitForTransition',
 
     this.testObject.waitForTransition(target, callback);
 
+    var transitionEvents = target.one.args[0][0];
```
Comment (alexindigo) [inline test/test-waitForTransition.js]: `target.one.getCall(0).args[0]` – would be more descriptive.

**Comment by alexindigo** [comment]
Thank you for the PR, looks awesome. Couple things to make it perfect and it's good to go.

File: test/test-waitForTransition.js (hunk 2 above)
Comment (abhisheksavant) [inline test/test-waitForTransition.js]: +1

File: test/test-waitForTransition.js (hunk 1 above)
Comment (abhisheksavant) [inline test/test-waitForTransition.js:43]: You mean trigger `one`, to check if `off` is called only once? coz if we repeat target.one.getCall(0).callArg(1), off will be called more than once. Sorry just wanted to be a bit more clear, may be misunderstood.

Comment (alexindigo) [inline test/test-waitForTransition.js:43]: Yes, cause no real jquery :)

**Pull request merged**